### ISSUE TITLE
RPC login and password in logs

### ISSUE THREAD

**Issue by romainPellerin** [gandrewstone/bitcoin#1463]
Hi BU team,

BU node 1.15.0.1-unk is writting RPC user and password, not sure if it's intended, does not seem appropriate to me, should be removed from the logger:

```2018-11-10 19:44:43 Bitcoin version v1.5.0.1-unk (Nov  6 2018, 10:30:00)
2018-11-10 19:44:43 Command line: /usr/bin/bitcoind -daemon -rest -conf=/home/bitcoin/.bitcoin/bitcoin.conf -pid=/run/bitcoind/bitcoind.pid 
2018-11-10 19:44:43 Single entry arguments:
2018-11-10 19:44:43         -addnode='35.162.151.77:18333'
2018-11-10 19:44:43         -conf='****.bitcoin/bitcoin.conf'
2018-11-10 19:44:43         -daemon=''
2018-11-10 19:44:43         -dbcache='16384'
2018-11-10 19:44:43         -maxconnections='20'
2018-11-10 19:44:43         -par='0'
2018-11-10 19:44:43         -pid='/run/bitcoind/bitcoind.pid'
2018-11-10 19:44:43         -regtest='0'
2018-11-10 19:44:43         -rest=''
2018-11-10 19:44:43         -rpcallowip='0.0.0.0/0'
2018-11-10 19:44:43         -rpcbind='0.0.0.0'
2018-11-10 19:44:43         -rpcpassword='passabcdefghijklmn123'
2018-11-10 19:44:43         -rpcuser='userabcdefghijklmn123'
2018-11-10 19:44:43         -server='1'
2018-11-10 19:44:43         -testnet='0'
2018-11-10 19:44:43         -txindex='1'```

Keep up the good work!

**Comment by ptschip** [comment]
thanks for reporting that, will put up a pr tomorrow.

**Comment by awemany** [comment]
Sorry, that was my screw-up with the log debug stuff. It is fixed in #1465 by @ptschip (thank you!)

**Comment by sickpig** [comment]
#1465 fix this issue. it is included in 1.5.1rc2 which become final in the beginning of the next week.
